public/components/incontext_insight/generate_popover_body.tsx

```diff
@@ -338,12 +357,7 @@ export const GeneratePopoverBody: React.FC<{
             <MessageActions
               contentToCopy={summary}
```

Comment:
Nit: Not related to this PR, it seems the only difference between these two `MessageActions` renders is `contentToCopy`. To keep the copied content the same as what is displayed, I'm wondering if we can add an option in `renderInnerFooter`. This way, we wouldn't need to check which message actions to be displayed.

Reply:
Done, thanks for the suggestion so that we can remove the duplicate code